es-vue-base/src/lib-components/EsNavBarSearchBar.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="align-items-center w-100 d-flex justify-content-center position-relative">
+        <div class="d-flex align-items-center mx-auto justify-content-center w-100">
+            <es-form-input
```

Comment:
With ESDS v2, I believe design system components including `EsFormInput` and `EsButton` and `IconSearch` need to be imported to be usable in downstream repos, even though they work in the docs